tests/src/texture/hipTextureRef2D.cpp

```diff
@@ -78,7 +80,7 @@ void runTest(int argc, char **argv)
 #ifdef __HIP_PLATFORM_HCC__
     hipLaunchKernelGGL(tex2DKernel, dim3(dimGrid), dim3(dimBlock), 0, 0, dData, tex.textureObject, width, height);

```

Comment:
Same here, can we use global instead ?

Reply:
These files shows examples for Texture runtime APIs. I updated the porting guide for runtime texture support too, therefore updated these files too for sync.

Reply:
Some recent version of HCC is supposed to support platform globals as well so we should be able to use texture globals for this case as well.  Perhaps that already works?  If not - @AlexVlx which version of HCC supports this?

Reply:
@bensander also note that post #246 global support will no longer be tied to HCC, so you may want to take that into account and schedule validation for this bit in accordance.

Reply:
Yes , post merging of #246 , I will update these runtime API example cases.